openvpn/management.go

```diff
@@ -123,7 +123,7 @@ func (management *Management) serveNewConnection(connection net.Conn) {
 func (management *Management) deliverLines() {
 	for {
 		line := <-management.lineReceiver
-		log.Debug(management.logPrefix, "Line delivering: ", line)
+		// log.Debug(management.logPrefix, "Line delivering: ", line)
```

Comment:
Dead code - we should remove it or use `log.Trace` for lower logging level.
Same applies for change at line 138.

Reply:
Fixed